.travis.yml

```diff
@@ -1,16 +1,13 @@
 language: node_js
 node_js:
-  - "4.1"
-  - "4.0"
+  - "7"
+  - "8"
```

Comment:
I also want to keep an older node version to test it there. Can you please add at least 5 and 6?